lib/cogctl/actions/relays/create.ex

```diff
@@ -0,0 +1,34 @@
+defmodule Cogctl.Actions.Relays.Create do
+  use Cogctl.Action, "relays create"
+  import Cogctl.Actions.Relays, only: [render: 3]
+
+  def option_spec do
+    [{:name, :undefined, 'name', {:string, :undefined}, 'Relay name (required)'},
+     {:token, :undefined, 'token', {:string, :undefined}, 'Relay token (required)'},
+     {:description, :undefined, 'description', {:string, :undefined}, 'Relay description'}]
+  end
+
+  def run(options, _args, _config, endpoint) do
+    params = convert_to_params(options, [name: :required,
+                                         token: :required,
+                                         description: :optional])
+    with_authentication(endpoint,
+                        &do_create(&1, params))
+  end
+
+  defp do_create(_endpoint, :error) do
```

Comment:
Minor style point: this error handling should be pulled up into the `run` function. That way, the logic around setting up and dealing with the parameters is confined to one place, rather than split. It simplifies `do_create/2` a bit, too, because it can just assume it's got valid parameters, rather than having to unwrap an `:ok` tuple.